packages/contracts-bedrock/deploy-config/getting-started.json

```
@@ -29,9 +29,16 @@
   "finalizationPeriodSeconds": 12,
 
   "proxyAdminOwner": "ADMIN",
+
   "baseFeeVaultRecipient": "ADMIN",
   "l1FeeVaultRecipient": "ADMIN",
   "sequencerFeeVaultRecipient": "ADMIN",
+  "baseFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
```

nit: split into a new group
```suggestion

  "baseFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
```